lib/ddtrace/contrib/action_cable/instrumentation.rb

```diff
@@ -52,7 +52,6 @@ def self.trace(channel, hook)
               configuration = Datadog.configuration[:action_cable]
 
               Datadog.tracer.trace("action_cable.#{hook}") do |span|
-                span.service = configuration[:service_name]
                 span.resource = "#{channel.class}##{hook}"
                 span.span_type = Datadog::Ext::AppTypes::WEB
 
```

Comment:
I'm curious: this whole block is very much like the `#process` of an event, but it's not modeled like that -- why?

Reply:
Is your question why this method is not yet another module that processes ActiveSupport events? If so, it's because this instrumentation path is not triggered through ActiveSupport events, thus have an explicit `Datadog.tracer.trace` call.

Reply:
I see... I think haha ;)